src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -934,7 +946,11 @@ public boolean verifyVisibilityMessageIsDisplayed(int questionNumber) {
     public boolean verifyVisibilityOptionsIsDisplayed(int questionNumber) {
         return getVisibilityOptions(questionNumber).isDisplayed();
     }
-    
+
+    public WebElement getVisibilityOptionTableRow(int questionNumber, int optionRowNumber) {
```

Comment:
@Sumei1009, if you are still reluctant to improve the selector to work on the non-displayed rows, then you will have to rename this method to indicate that only the displayed rows can be returned.